source/_integrations/anova_sous_vide.markdown

```diff
@@ -0,0 +1,42 @@
+---
+title: Anova Sous Vide
+description: Instructions on how to integrate Anova Wifi Sous Vide into home assistant.
+ha_category:
+  - Sensor
+ha_iot_class: Cloud Polling
+ha_release: 2023.2
+ha_codeowners:
+  - '@conway220'
+ha_domain: anova_sous_vide
+ha_integration_type: integration
+---
+
+The Anova Sous Vide sensor platform allows you to control [Anova](https://anovaculinary.com/pages/find-your-anova-precision-cooker) sous vides with wifi capability.
+
+Supported devices (tested):
+
+- [Anova Precision Cooker](https://anovaculinary.com/products/anova-precision-cooker)
+
+Supported devices (untested):
+
+- [Anova Precision Cooker Pro](https://anovaculinary.com/products/anova-precision-cooker-pro)
+- [(Anova)RED Precision Cooker](https://anovaculinary.com/products/anova-red-precision-cooker)
+- [(Anova)RED Precision Cooker Pro](https://anovaculinary.com/products/anova-red-precision-cooker)
+
+The 'nano' versions of the sous vide are not supported, but as long as your app is connected to the sous vide, the data should update. They would be better served using BLE instead of API calls.
+
+To add this platform to your installation, You will need your device id.
+
+You can find the Device id by connecting your sous vide to the app, Then going Profile -> Settings(Gear Icon) -> Connection -> Cooker Details.
```

Comment:
Cool, so how to add it to Home Assistant? That part is missing.

Be sure to include the config flow include :)

Reply:
I added the config flow include. Is it more like how it should be now? It can only be added with the userflow.